### PR TITLE
Reword depending on archives

### PR DESCRIPTION
**Pull request by wolfs**

### CODE REVIEW

File: contents/caching-java-projects.adoc [hunk 1]
```diff
@@ -56,12 +56,9 @@ integTest {
 
 ==== Archives as inputs
 
-Depending on a zip or tar archive as an input file will lead to cache misses since rebuilding will probably change the metadata in the archive as described in <<concepts_repeatable_task_outputs>>.
-To avoid this problem altogether you can instead depend on the contents on the archive. See also the section on dealing with <<volatile_outputs,non-repeatable outputs>>.
-
-Let us assume that you ship an executable jar as a distribution.
-Instead of adding the jar itself as an input to your test task, you should depend on whatever makes up the jar.
-And that would most likely be the classes and some resources.
+It is common for integration tests to depend your packaged application.
```
Comment (lptr) [inline contents/caching-java-projects.adoc]: Missing "the".

File: contents/caching-java-projects.adoc [hunk 2]
```diff
@@ -56,12 +56,9 @@ integTest {
...
+If this happens to be a zip or tar archive then adding it as an input to the integration test task will lead to cache misses since rebuilding will change the metadata in the archive as described in <<concepts_repeatable_task_outputs>>.
```
Comment (lptr) [inline contents/caching-java-projects.adoc]: Missing comma before "since", but would be nice to break this long sentence into two, too.

Comment (lptr) [inline contents/caching-java-projects.adoc]: Also "rebuilding" is too vague. You could talk about rebuilding the archive. And "will" is too strong here, as it's only a possibility: unless the original files have been changed you will get the same archive, byte-for-byte.

Comment (wolfs) [inline contents/caching-java-projects.adoc]: How about now?

**Pull request merged**